components/homme/src/share/namelist_mod.F90

```diff
@@ -219,9 +219,11 @@ subroutine readnl(par)
     character(len=80) :: errstr, arg
```

Comment:
These two files differ in e3sm and in scream, but this file needs to be reviewed before using scream version as is (i believe Andrew would know how this file should look like).

Reply:
Agreed. My impression is that these changes cause the upstream Homme standalone DP tests to DIFF. It's on my to-do list to find out why.

Reply:
homme_integration passes on Chrysalis when run from the SCREAM repo, so the HOMME DP tests that once were broken are now fine.

Reply:
Should we push those changes upstream in a separate E3SM PR before this PR goes back in, or wait and address in the big merge PR?

Reply:
I'm fine having these go in with the big PR since the mods don't affect homme_integration.